package.json

```diff
@@ -38,7 +38,6 @@
     "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
     "@types/react-router-dom": "^5.1.3",
-    "@types/segment-analytics": "0.0.32",
```

Comment:
package-lock should not be modified that much by this removal. Honestly, it's a type dependency that doesn't include any runtime code and weighs like <1kb, so feel free to keep it in the deps if getting package-lock to behave is proving too hard.

Reply:
I'm not sure how I should have done this. I just remove the line and did npm install :shrug:

Reply:
yeah, it's probably because this project was last built on npm6 and you're on npm7 -- it's not the end of the world, but not ideal

Reply:
Clearly that. I reverted it.